src/ios/Reporting.m

```diff
@@ -1,3 +1,4 @@
+#import <Webkit/Webkit.h>
```

Comment:
We can get rid of this import by dynamically calling the WKWebView Javascript executor if the object responds to the selector.